### PR TITLE
Python 3 Compatibility

### PR DESCRIPTION
**Pull request by mariorz**
Includes PRs 
- https://github.com/bitsoex/bitso-py/pull/11
- https://github.com/bitsoex/bitso-py/pull/12
- https://github.com/bitsoex/bitso-py/pull/16

All tests passing for python 2/python 3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -402,7 +409,7 @@ u'3CEWgs1goBbafUoThjWff4oX4wQKfxqpeV'
 ##         - string
 ## address - The Bitcoin address to send the amount to
 ##         - string
->>> api.btc_withdrawal('14', '0x55f03a62acc946dedcf8a0c47f16ec3892b29e6d')
+>>> api.btc_withdrawal('0.14', '1TVXn5ajmMQEbkiYNobgHVutVtMWcNZGV')
```
Comment (vogelito) [inline README.md]: Perhaps it would be better to specify a smaller amount? E.g. `0.000000001`

File: README.md [hunk 2]
```diff
@@ -414,7 +421,7 @@ ok   # Returns 'ok' on success
 ##         - string
 ## address - The Bitcoin address to send the amount to
```
Comment (vogelito) [inline README.md]: I believe this should read `The ether address to send the amount to`

File: README.md [hunk 3]
```diff
@@ -414,7 +421,7 @@ ok   # Returns 'ok' on success
...
 ##         - string
->>> api.eth_withdrawal('1.10', '1TVXn5ajmMQEbkiYNobgHVutVtMWcNZGV')
+>>> api.eth_withdrawal('1.10', '0x55f03a62acc946dedcf8a0c47f16ec3892b29e6d')
```
Comment (vogelito) [inline README.md]: should this amount be smaller just in case? Who do these addresses belong to BTW?

File: bitso/api.py [hunk 4]
```diff
@@ -77,8 +85,15 @@ def __init__(self, key=None, secret=None):
...
-        self.base_url_v2 = "https://bitso.com/api/v2"
-        self.base_url = "https://bitso.com/api/v3"
+        
+        if dev:
+            self.base_url_v2 = "https://dev.bitso.com/api/v2"
+            self.base_url = "https://dev.bitso.com/api/v3"
+    
+        else:
+            self.base_url_v2 = "api.bitso.com/v2"
+            self.base_url = "api.bitso.com/v3"
```
Comment (vogelito) [inline bitso/api.py]: any reason why the dev addresses are prefixed with `https` but these are not?

Comment (mariorz) [reply bitso/api.py]: no, that's wrong. good catch

File: README.md [hunk 5]
```diff
@@ -419,9 +419,9 @@ ok   # Returns 'ok' on success
 ## Triggers a bitcoin withdrawal from your account
```
Comment (CrispinFlowerday) [inline README.md]: This should mention ethereum as well (not bitcoin)

File: README.md (hunk 2 above)
Comment (mariorz) [reply README.md]: This is the bitcoin_withdrawal method, but fixed this elsewhere 👍

File: README.md [hunk 6]
```diff
@@ -412,9 +419,9 @@ ok   # Returns 'ok' on success
...
 ## amount  - The amount of BTC to withdraw from your account
```
Comment (CrispinFlowerday) [inline README.md]: Oh, and this guy should be "ETH"

File: README.md (hunk 5 above)
Comment (mariorz) [reply README.md]: yup, thx

File: README.md (hunk 6 above)
Comment (mariorz) [reply README.md]: 👍

File: bitso/api.py [hunk 7]
```diff
@@ -24,18 +24,26 @@
...
+  basestring
+except NameError:
+  basestring = str
 
-from bitso import (ApiError, ApiClientError, Ticker, OrderBook, Balances, Fees, Trade, UserTrade, Order, TransactionQuote, TransactionOrder, LedgerEntry, FundingDestination, Withdrawal, Funding, AvailableBooks, AccountStatus, AccountRequiredField)
+try:
+    from urllib.parse import urlparse, urlencode
+except ImportError:
+    from urlparse import urlparse
+    from urllib import urlencode
```
Comment (JuIioCesar) [inline bitso/api.py]: @mariorz  Why don't we use the python 2 & 3 code instead? Here's a python 2, 3 cheat sheet 
Please see the `Imports relative to a package` section and the `base string` sections as a starter
https://goo.gl/87qGVS

Comment (mariorz) [reply bitso/api.py]: Becase this works just as well.

Comment (mariorz) [reply bitso/api.py]: Don't really agree with the requested changes, so closing this PR.

**Pull request closed without merging**